chain/client/src/client.rs

```diff
@@ -76,7 +89,13 @@ impl ClientActor {
         network_actor: Recipient<NetworkRequests>,
         block_producer: Option<BlockProducer>,
     ) -> Result<Self, Error> {
+
         // TODO: Wait until genesis.
+
+         //think about it
+        let _ = wait_until_genesis(&genesis_time)?;
```

Comment:
don't need `let _ =`
and think about it -> Waiting until genesis is started early.

Reply:
nvm, you do need that.